src/startup/routes.jsx

```diff
@@ -1,7 +1,12 @@
 import { HomePage } from 'home'
+import { DetectionPage } from 'detection'
 import { About } from 'about'
 
 export const routes = [{
+  path: '/detection/:id',
+  Component: DetectionPage,
```

Comment:
can you make Component -> component

Reply:
![image](https://user-images.githubusercontent.com/23569938/125487372-1f0cbef8-217c-4132-ad26-8cf0d424adbd.png)
Not because of this

Reply:
Checked the documentation. He is right. 
https://reactrouter.com/web/example/route-config

Reply:
done